### PR TITLE
Jest Prototype results

### LINKED ISSUE #6: Add documentation for testing prototype
This should detail what the prototype was used for and the results.

### PR DESCRIPTION
**Pull request by ets994**
Closes #6

Just a quick overview of getting Jest set-up and working.

### CODE REVIEW

**Comment by braunsonm** [comment]
For example, if you navigate here: https://github.com/UniversityOfSaskatchewanCMPT371/term-project-fall2019-team-2/tree/master/prototypes/bootcamp

You can immediately see the README at the bottom properly formatted because it's named README.md and uses the template.

**Comment by ets994** [comment]
Okay, the stuff has been changed over to be the README.md for the prototype directory.

**Comment by eileenvanheerde** [comment]
Looks good!

File: prototypes/testing-stack/ets994/test-app/README.md [hunk 1]
```diff
@@ -1,68 +1,24 @@
...
-This section has moved here: https://facebook.github.io/create-react-app/docs/advanced-configuration
-
-### Deployment
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/deployment
-
-### `npm run build` fails to minify
-
-This section has moved here: https://facebook.github.io/create-react-app/docs/troubleshooting#npm-run-build-fails-to-minify
+- I couldn't get Jest to work without also installing babel.
```
Comment (eileenvanheerde) [inline prototypes/testing-stack/ets994/test-app/README.md]: You were using Visual Studio when you did this prototype, right? Jest is incredibly easy to set with Webstorm.

**Pull request merged**